torch/utils/_triton.py

```diff
@@ -1,10 +1,14 @@
 # mypy: allow-untyped-defs
 import functools
 import hashlib
+import importlib
 
 
 @functools.lru_cache(None)
 def has_triton_package() -> bool:
+    triton_spec = importlib.util.find_spec("triton")
```

Comment:
catch the attribute error and return False instead

Reply:
@oulgen Thanks for review.
Just change to use AttributeError check on triton.language.dtype